packages/web3-plugin-example/test/unit/custom_rpc_methods.test.ts

```diff
@@ -0,0 +1,68 @@
+/*
+This file is part of web3.js.
+
+web3.js is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+web3.js is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
+*/
+import Web3 from 'web3';
+
+import { CustomRpcMethodsPlugin } from '../../src/custom_rpc_methods';
+
+describe('CustomRpcMethodsPlugin Tests', () => {
+	it('should register CustomRpcMethodsPlugin plugin', () => {
+		const web3 = new Web3('http://127.0.0.1:8545');
+		web3.registerPlugin(new CustomRpcMethodsPlugin());
+		// Both CustomRpcMethodsPlugin and ContractMethodWrappersPlugin
+		// redeclare the web3 module, and this seems to confuse the TypeScript server
+		// @ts-expect-error  Unsafe call of an `any` typed value
```

Comment:
As mentioned in the comment, I believe this is caused by both plugin classes re-declaring the `web3` module:

```typepscript
declare module 'web3' {
	interface Web3 {
		pluginNamespace: pluginClass;
	}
}
```

As commenting out the `ContractMethodWrappersPlugin` class causes these errors to disappear